main.py

```diff
@@ -0,0 +1,95 @@
+from InstagramAPI import InstagramAPI
+import datetime
+import itertools
+from sqlalchemy import create_engine
+from sqlalchemy.orm import sessionmaker
+
+from models import Base, InstagramUsers
+
+user_name = 'case_iphone_lviv2019'
+password = 'игвяеуіе123'
+
+
+class GetInstagramInfo:
+    def __init__(self, username, password):
+        self.api = InstagramAPI(username=username, password=password)
+        self.api.login()
+
+    def get_user_id(self, name):
+        """Returns id user."""
+        self.api.searchUsername(name)
+        return self.api.LastJson["user"]["pk"]
+
+    def get_followers(self, user_id):
+        """Returns list of user's followers."""
+        followers = self.api.getTotalFollowers(user_id)
+        return [follower['username'] for follower in followers]
+
+    def get_followings(self, user_id):
+        """Returns list of user's followings."""
+        followings = self.api.getTotalFollowings(user_id)
+        return [following['username'] for following in followings]
+
+    def save_db(self, followers, followings):
+        engine = create_engine(f'sqlite:///users.db')
+        Base.metadata.create_all(engine)
+        Session = sessionmaker(bind=engine)
+        session = Session()
+        now = datetime.datetime.now().strftime('%Y-%m-%d %H:%M')
+
+        for (followings, followers) in itertools.zip_longest(followings, followers, fillvalue=''):
+            text = InstagramUsers(followings=followings, followers=followers, status="new", time=str(now))
+            session.add(text)
+            session.commit()
+
+    def user(self):
+        engine = create_engine(f'sqlite:///users.db')
+        Base.metadata.create_all(engine)
+        Session = sessionmaker(bind=engine)
+        session = Session()
+
+        now = datetime.datetime.now().strftime('%Y-%m-%d %H:%M')
+
+        database_followings = session.query(InstagramUsers.followings).all()
+        database_follower = session.query(InstagramUsers.followers).all()
+
+        followings_list = [y for x in database_followings for y in x]
+        followers_list = [y for x in database_follower for y in x]
+
+        for following in followings:  # це  в бд людей на яких я підписався недавно
```

Comment:
Where did you get this variable `followings`?

Reply:
done